vsintegration/src/fsharp.vsreferences.targets

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Again, let's please avoid needlessly adding more and more files. We have targets files already that all projects import, why can't we use those?